.gitignore

```diff
@@ -0,0 +1,2 @@
+development.db
+Gemfile.lock
```

Comment:
❌  `Gemfile.lock` should always be committed to the repo. Otherwise, `bundle install` might not always install the same Gem versions.